packages/server/src/routes/me/mint.ts

```diff
@@ -139,10 +155,9 @@ export const mint = (
       const bigAmount = new Big(amount).round(2, 0);
 
       let mintData;
-      const mintType = call.body.type;
-      if (mintType === MintType.BASIC_INCOME) {
+      if (call.body.type === MintType.BASIC_INCOME) {
         mintData = _mintBasicIncome(loggedInMember, bigAmount);
-      } else if (mintType === MintType.REFERRAL_BONUS) {
+      } else if (call.body.type === MintType.REFERRAL_BONUS) {
```

Comment:
@markulrich I reverted this because TypeScript was unable to resolve the body params without directly checking